app/views/comments/_form.html.erb

```diff
@@ -100,15 +100,15 @@
 <% end %>
 
 <% if @comment.persisted? %>
-  <script defer>
-    var editForm = document.getElementById("edit_comment_<%= @comment&.id %>");
+  <script>
+    const editForm = document.getElementById("edit_comment_<%= @comment&.id %>");
```

Comment:
Refactor (blocking): @jmau111, for now let's remove the `defer` attributes only. Making the variable declarations `const` over `var` will be problematic as all of those variables are global and could break other things in the app as not all code is bundled with webpack at the moment.

Here is an example of it in action failing.

![image](https://user-images.githubusercontent.com/833231/126494025-6fced221-1229-40b6-bc01-559b424d72ef.png)

Reply:
@nickytonline are you sure? can't see any other usage in the entire base code of `editForm`

Reply:
done in https://github.com/forem/forem/pull/14234/commits/9ecae0622c50a62e872e093e2ba97bdcacbb8bdd, I guess  the benefit to risk equation is not high enough to change for `const`